src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="calendar calendar--month-length--29">
+      <div class="calendar__day calendar__day--start--sun"></div>
```

Comment:
you should use class with day of week for calendar, not day
move only first square to move all of them